products/spring-boot.md

```diff
@@ -99,9 +109,14 @@ See [Spring Boot Milestones page](https://github.com/spring-projects/spring-boot
 upcoming releases and [Spring Boot Support page](https://spring.io/projects/spring-boot#support) for
 more details about support roadmap.
 
-- Spring Boot 3.x requires **at least a Java 17 runtime**,
-- Spring Boot 2.7.5 (and therefore the latest Spring Framework 5.3.23) supports Java 19 while also
-  remaining compatible with Java 11 and 8.
+Spring Boot Java compatibility is the following:
```

Comment:
```suggestion
## Java Compatibility
```

Reply:
I find title in a product description a little bit strange in regards to the "More information is available on..." paragraph.

Shouldn't we add `<hr>` before "More information is available on..." to make it clear it does not relate to the last section ?

Reply:
I think I'd tried this and it didn't look nice. Can file another PR for this.

Reply:
Done, `hr` will be discussed in another PR.